deploy/ansible/roles/docker/tasks/main.yml

```diff
@@ -1,3 +1,27 @@
 ---
-  - include_tasks: ubuntu.yml
-    when: ansible_distribution == 'Ubuntu'
+
+- name: remove existing python modules
+  pip:
+    name: "{{ item }}"
+    state: absent
+  with_items:
+    - docker
+    - docker-py
+    - docker-compose
+
+- name: install docker python modules
+  pip:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - docker
+    - docker-compose
+
+
+- include_tasks: install_docker.yml
```

Comment:
You can safely omit the check for Linux distribution. In this and following playbooks assume it is Ubuntu Linux as we use it as the primary distro. We also gradually remove these checks in our playbooks. YAML files in `tasks` can also be simply named `main.yml` (not `ubuntu.yaml` included into `main.yml` as previously)

Reply:
Sometimes I use distro, other than ubuntu